test/remote/gateways/remote_merchant_warrior_test.rb

```diff
@@ -8,11 +8,19 @@ def setup
     @failure_amount = 205
 
     @credit_card = credit_card(
-      '5123456789012346',
-      :month => 5,
-      :year => Time.now.year + 2,
-      :verification_value => '123',
-      :brand => 'master'
+      '4564710000000004',
+      :month => 2,
```

Comment:
```suggestion
      :month => '2',
```
It seems like Rubocop doesn't like having a bare `05`. Test seem to all still pass w quoted values.